src/math/src/tests/wad_ray_math_test.cairo

```diff
@@ -0,0 +1,160 @@
+use core::debug::PrintTrait;
```

Comment:
```suggestion
```

Reply:
I thought that scarb format will run on pre-commit. It's removed

Reply:
scarb format only formats it doesn't remove unused imports. Also i don't think we have any pre commit hook on this repo do we ?

Reply:
Running `scarb format` locally had fix import order at least. Didn't really check if useless import has been removed